File: Python developer/step04/4.1.04.py
'''

Ваша программа получает на вход целое число n. В случае если оно четное – прибавьте к нему 10, а если нечетное отнимите 10.

В качестве вывода ваша программа должна выдать результат вычислений.

Sample Input 1:

10

Sample Output 1:

20

Sample Input 2:

9

Sample Output 2:

-1

Sample Input 3:

0

Sample Output 3:

10

'''

n = int(input())
if n%2==0:
    print(n+10)
else:
    print(n-10)

File: Fundamentals of C programming/step04/4.1/4.1.04.c
/*

Составьте программу подсчёта размера оплаты за электроэнергию по показаниями счётчика и тарифу. На вход поступают 3 числа.
Первое число -- целое число. Показания счётчика в кВт*ч на начало месяца.
Второе число -- целое число. Показания счётчика в кВт*ч на конец месяца.
Третье число -- вещественное число. Стоимость одного кВт*ч в рублях.

Программа должны вывести на экран размер оплаты за электроэнергию. Результат выведите с двумя знаками после запятой.

Sample Input:

1200 1400 0.2

Sample Output:

40.00

*/

#include <stdio.h>
int main() {
  int a, b;
  double c, summa;
  scanf("%d",&a);
  scanf("%d",&b);
  scanf("%lf",&c);

  summa = (b-a)*c;
  printf("%.2lf\n",summa);

  return 0;
}
